lib/bloc/present_signed_document_state.dart

```diff
@@ -44,13 +44,24 @@ class PresentSignedDocumentErrorState extends PresentSignedDocumentState {
   }
 }
 
-class PresentSignedDocumentSuccessState extends PresentSignedDocumentState {
+class PresentSignedLocalDocumentSuccessState
+    extends PresentSignedDocumentState {
   final File file;
 
-  const PresentSignedDocumentSuccessState(this.file);
+  const PresentSignedLocalDocumentSuccessState(this.file);
 
   @override
   String toString() {
     return "$runtimeType(file: $file)";
   }
 }
+
+class PresentSignedRemoteDocumentSuccessState
+    extends PresentSignedDocumentState {
+  const PresentSignedRemoteDocumentSuccessState();
+
+  @override
+  String toString() {

```

Comment:
Tento override uz netreba, nakolko je zdedeny z parent `PresentSignedDocumentState` type.
Len tie, co maju daco navyse by to mali overridovat.